text/0000-ember-language-tooling-roadmap.md

```diff
@@ -0,0 +1,169 @@
+---
+stage: accepted
+start-date: 2024-03-17
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams:
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: # Fill this in with the URL for the Proposal RFC PR
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Ember Language Tooling Roadmap
+
+## Summary
+
+This RFC provides a roadmap towards modernizing, unifying, and improving the Ember language tooling ecosystem. In particular, we recommend the following long term goals:
+
+1. Remove support of and any reference to GlimmerX from Glint
```

Comment:
This appears to have happened in https://github.com/typed-ember/glint/pull/716/

Reply:
Yes; I'm feeling based on your feedback and others, including former Glint maintainers, that going through the full RFC process for this probably doesn't make much sense. I will probably close it shortly (though I'll continue to use the roadmap).

Reply:
with above in mind, maybe worth to convert to an issue under https://github.com/emberjs/tracking-polaris/issues